internal/core/graphics/image.rs

```diff
@@ -435,6 +435,20 @@ impl ImageInner {
             _ => None,
         }
     }
+
+    /// Returns true if the image is an SVG (either backed by resvg or HTML image wrapper).
+    pub fn is_svg(&self) -> bool {
+        match self {
+            Self::Svg(_) => true,
+            #[cfg(target_arch = "wasm32")]
+            Self::HTMLImage(html_image)
+                if html_image.dom_element.current_src().ends_with(".svg") =>
```

Comment:
This line could be made into a function  `html_image.is_svg()`  so there is only one place with this .svg logic.  I wonder if there are no more reliable way to detect if an image is svg though  (mime type?)

Reply:
Good idea, done (the function part).

I wish, too, that there was a more reliable way. don't know of any though :(